packages/admin-panel/src/pages/resources/ResourcePage.js

```diff
@@ -83,4 +86,5 @@ ResourcePage.defaultProps = {
   ExportModalComponent: null,
   onProcessDataForSave: null,
   baseFilter: {},
+  defaultSorting: null,
```

Comment:
I guess `[]` will not work with the existing code? (haven't had a look)

Reply:
Actually that's a great point, and changing this here makes the table initialisation much cleaner 🎉